### ISSUE TITLE
We have violation of service notice from Medium

### ISSUE THREAD

**Issue by ZhymabekRoman** [Freedium-cfd/web#9]
Hello,
 
We are writing to notify you that your Medium.com account is in violation of our rules, and your profile and posts will no longer be publicly available on Medium.
 
https://policy.medium.com/paid-terms-of-service-cc7f8e165178
 
https://freedium.cfd/
 
Your work will remain accessible to you while signed in. Your content may be exported at any time by going to your Settings (https://medium.com/me/settings). From there, go to to Security and apps and click Download your information > Export to export your data, which includes your published stories.
 
Your Medium membership, if you have one, will be canceled.
 
Thanks,
Trust & Safety

**Comment by ZhymabekRoman** [comment]
Here we go, again...

**Comment by ZhymabekRoman** [comment]
Hello,
 
We are writing to notify you that your Medium.com account is in violation of our rules, and your profile and posts will no longer be publicly available on Medium.
 
https://medium.com/policy/medium-rules-30e5502c4eb4
 
Facilitation or evidence of violating the terms of service of Medium or third party(s)
 
https://github.com/Freedium-cfd
 
Your work will remain accessible to you while signed in. Your content may be exported at any time by going to your Settings (https://medium.com/me/settings). From there, go to to Security and apps and click Download your information > Export to export your data, which includes your published stories.
 
Your Medium membership, if you have one, will be canceled.
 
Thanks,
Trust & Safety

**Comment by HakaishinShwet** [comment]
freedium.cfd is still working have they banned your account sir? and if yes then how is freedium still working is question running in my mind :-) .

**Comment by ZhymabekRoman** [comment]
> freedium.cfd is still working have they banned your account sir?

Yes, twice, same account. I just pay for the subscription again and everything still works.

> what will be the future of freedium if these things happen again

There are currently two ways to get around Medium's paywall: the first is legal - pay for a subscription, the second is (semi-)legal - use Google WebCache. Problem with WebCache that new posts are not available

**Comment by tejush1998** [comment]
Looks like its down again :')

**Comment by Wesley-waka** [comment]
Yeah the site is down :')

**Comment by ZhymabekRoman** [comment]
Fixed, thanks!

**Comment by isaiah2004** [comment]
Well look like they stopped being lazy and actually patched it.

**Comment by ZhymabekRoman** [comment]
Workaround done, tomorrow will be released (12-24 hour)

**Comment by ZhymabekRoman** [comment]
Upd: Fixed, just left disk space on the server lmao

**Comment by ZhymabekRoman** [comment]
Please use #16 issue to report any problems

**Comment by ZhymabekRoman** [comment]
Hello,

Cloudflare received a DMCA copyright infringement complaint regarding: freedium.cfd

The information we received was the following:

Reporter's Name: Mihaita Tinta
Reporter's Email Address: [mihaita.tinta@gmail.com](mailto:mihaita.tinta@gmail.com)
Reporter's Title: Freedium is copying my articles from medium.com
Reporter's Address: Pajurei 2B, Bucharest , Bucharest, RO

Reported URLs:
hxxps://freedium[.]cfd/9faeaf0a630e


Original Work: My articles are published on medium.com. For example: https://medium.com/javarevisited/a-practical-approach-to-integration-tests-9faeaf0a630e#bypass

The whole idea with freedium is to take advantage of other peoples work so the site owner gets easy money via patreon or donations.

Can you stop this? 


We have forwarded this complaint to your hosting provider. 

To respond to this issue, please reply to [abusereply@cloudflare.com](mailto:abusereply@cloudflare.com).

Thanks,
The Cloudflare Team
